game.js

```diff
@@ -547,69 +547,60 @@ module.exports = function createGame(options) {
         return lodash.intersection(state.roles, lodash.flatten([roles]))[0];
     }
 
-    function command(playerIdx, command) {
-        debug('command from player: ' + playerIdx);
-        debug(command);
-        var i, action, message;
-        var playerState = state.players[playerIdx];
-        if (playerState == null) {
-            throw new GameException('Unknown player');
+    const playerIsReady = (command, playerIdx) => {
+        if (state.players[playerIdx].isReady !== true) {
+            throw new GameException('You cannot start the game');
```

Comment:
You are calling this function in many other cases, not just when someone tries to start the game. Might be better to use a message related to not being ready. To be honest, I cannot remember why it is important to check that a player is ready before he starts a game.